### PR TITLE
[DOCFIX] Update performance tuning docs

### PR DESCRIPTION
**Pull request by gpang**

### CODE REVIEW

**Comment by gpang** [comment]
@ZacBlanco

File: docs/en/operation/Performance-Tuning.md [hunk 1]
```diff
@@ -195,7 +192,8 @@ There are 3 options for loading a missing path: `Never`, `Once`, `Always`.
 `ONCE` will use the default behavior of only scanning each directory once ever, and `NEVER` will never consult the UFS
 and thus prevent Alluxio from scanning for new files at all.
 
-The Alluxio master maintains a cache to approximate which UFS paths have been previously loaded, to approximate the `Once` behavior.
+The Alluxio master maintains a cache to approximate which UFS paths have been previously loaded,
+to approximate the `Once` behavior.
```
Comment (ZacBlanco) [inline docs/en/operation/Performance-Tuning.md]: `to approximate` twice in the same sentence sounds a little odd here. Can we try re-wording?

File: docs/en/operation/Performance-Tuning.md [hunk 2]
```diff
@@ -320,8 +318,8 @@ alluxio.master.persistence.blacklist=.staging,_temporary
...
 persist.
 This works because eventually these temporary files will be deleted or renamed to permanent files.
-Because `alluxio.user.file.persist.on.rename=true` is set, the files will be considered for persist
-again when renamed.
+Because `alluxio.user.file.persist.on.rename=true` is set, the files will be considered for
+persistence again when renamed.
 Note that persist on rename works for directories as well as files - if a top level directory is
 renamed with the persist on rename option, all its not yet persisted files will be scanned for
-eligbility for persist.
+eligibility for persist.
```
Comment (ZacBlanco) [inline docs/en/operation/Performance-Tuning.md]: "...will be scanned for eligibility for persist" sounds kind of odd.

How do you feel about: "any files underneath the top-level directory will be considered for persistence"
or
"any files underneath the top-level directory will be considered for persist eligibility"

File: docs/en/operation/Performance-Tuning.md (hunk 1 above)
Comment (gpang) [reply docs/en/operation/Performance-Tuning.md]: updated

File: docs/en/operation/Performance-Tuning.md (hunk 2 above)
Comment (gpang) [reply docs/en/operation/Performance-Tuning.md]: updated

**Comment by gpang** [comment]
alluxio-bot, merge this please

**Comment by gpang** [comment]
alluxio-bot, cherry-pick this to branch-2.2 please

**Pull request merged**